### PR TITLE
[MRG] Traditional ML Converter using hummingbird-ml

### LINKED ISSUE #128: convert improvement
#### Software and Hardware Versions 
ModelCI v1.x.x, CUDA Version vx.x.x, GPU device used...

#### Problem description

We may try to adopt https://github.com/microsoft/hummingbird to our system

#### Expected Behavior

Supported more conversions

#### Other Information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

### PR DESCRIPTION
**Pull request by YuanmingLeee**
Closes #128 

TODO:
- [x] Traditional ML -> pytorch
- [x] Traditional ML -> ONNX
- [x] Update doc

#### Does this change require updates to documentation?
Yes

#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

**Comment by HuaizhengZhang** [comment]
Are the document and issues ready?

**Review by HuaizhengZhang — DISMISSED** [review]

**Comment by HuaizhengZhang** [comment]
I have approved the PR.

Please update the document, open issues, and prepare an update log to record our upgrading.

**Comment by HuaizhengZhang** [comment]
The CI issue should also be fixed

**Review by HuaizhengZhang — APPROVED** [review]
Update changelog.md before merging

File: tests/test_converter.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+
+if __name__ == '__main__':
+    # test_xgboost_to_onnx()
+    # test_lgbm_to_onnx()
+    # test_sklearn_to_onnx()
+    # test_xgboost_to_torch()
+    # test_lightgbm_to_torch()
+    # test_sklearn_to_torch()
+    test_onnx_to_pytorch()
```
Comment (huangyz0918) [inline tests/test_converter.py:134]: 这些在 CI 环境里面能不能跑起来，能跑就不用注释了 @YuanmingLeee

Comment (YuanmingLeee) [reply tests/test_converter.py:134]: 不影响CI

**Review by huangyz0918 — APPROVED** [review]
lgtm

**Pull request merged**